handler/grpc/middleware/auth.go

```diff
@@ -65,15 +63,20 @@ func getTokenValue(headers metadata.MD) string {
 	return token
 }
 
-func verifyToken(ctx context.Context, tokenString string) (string, []string, error) {
-	redis := redis.Connection()
-	authClient := userService.NewAuthClient(os.Getenv("JWT_SECRET"), redis)
-	claims, err := authClient.ValidateToken(ctx, tokenString)
+func verifyToken(ctx context.Context, token string) (string, []string, error) {
```

Comment:
could we name the responses, not only (string, []string, error) so We can read this function easily.

Reply:
done